### PR TITLE
RPRBLND-1822: Baikal Next support

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Implement Baikal Next support.

### EFFECT OF CHANGE
Added render quality HybridPro.

### TECHNICAL STEPS
Name convention BaikalNext as HybridPro
Implementation based on Hybrid
Added HybridPro to render property panel in quality section
Temporary added file HybridPro.dll
Temporary added file HybridPro.so
Added new export_hybrydpro() for nodes, temporary uses export_hybryd()
Added new modules animation_engine_hybridpro.py, viewport_engine_hybridpro.py, render_engine_hybridpro.py, context_hybridpro.py
Added class registration for RenderEngineHybridPro, ViewportEngineHybridPro, AnimationEngineHybridPro
Added various additional checks for HybridPro

### NOTES FOR REVIEWERS
Need to be updated at least to core 2.2.9

### CODE REVIEW

File: src/rprblender/properties/render.py [hunk 1]
```diff
@@ -455,6 +456,11 @@ def is_tile_render_available(self):
             ('LOW', "Low", "Low render quality"),
         ]
 
+    if pyhybridpro.enabled:
+        render_quality_items += [
+            ('HYBRIDPRO', "HybridPro", "HYBRIDPRO render quality using BaikalNext, only hardware ray tracing support")
```
Comment (bsavery) [inline src/rprblender/properties/render.py:461]: Name should be RPR Interactive
Northstar is now named "RPR Final"

**Review by bnagirniak — APPROVED** [review]
Added some improvements to naming and disabled hybrid render qualities if hybridpro when hybridpro is enabled.

**Review by DagerD — CHANGES_REQUESTED** [review]
I still have "Interactive" option in Render Quality and in doesn't work for me (5700 xt).
image

**Comment by bsavery** [comment]
> I still have "Interactive" option in Render Quality and in doesn't work for me (5700 xt).
> image

RPR - Interactive is only supported on GPUs with Hardware raytracing.  

We should still keep low - medium -high

**Review by DagerD — APPROVED** [review]
Fixed.

**Comment by VascoPi** [comment]
Problems with core2.2.10.
These two pictures were made with core2.2.9 and 2.2.10(failed), no errors. Waiting responde from HybridPro developers.
image
blender_attmyy9wxr_720

**Comment by bnagirniak** [comment]
I confirm, with new core all materials in HybridPro are pink now, seems usual materials aren't supported now.
image

**Comment by bsavery** [comment]
The should be supported.

**Review by bnagirniak — APPROVED** [review]
Tested - now HybridPro works as expected.

**Pull request merged**